src/main/webapp/header.jsp

```diff
@@ -13,6 +13,7 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 --%>
+<!-- NOTE: register.jsp and chat.jsp do not use the header -->
```

Comment:
I would leave this out of the JSP or put it into <%-- comments instead. In particular, this will show up in people's browsers if they do "view source" and generally you want to avoid leaking details like this.